### DISCUSSION TITLE
I cannot apply for the student beneift

### DISCUSSION THREAD

**Discussion by QThans** [community/community#60765] [GitHub Education]
### Hi everyone,

I am a current student from Universiti Sains Malaysia. But I am not in Malaysia and return to my home country for family things, maybe need a long time. So How can I apply for this?

**Comment by yaxan** [comment]
How is being out of the country preventing you from applying? Is it not just an online application?

**Reply by QThans** [reply]
It needs me to take a picture by using the camera, which must contain my name, school name and so on.
![image](https://github.com/community/community/assets/18275400/994f894d-c7ed-4018-978d-3d6439a3c7db)

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
